react-native-libraries.json

```
@@ -8397,10 +8397,22 @@
     "expo": true
   },
   {
-    "githubUrl": "https://github.com/Luffos/rxn-keylistener",
+    "githubUrl": "https://github.com/Luffos/rxn-input",
     "android": true,
     "web": true
   },
+  {
+    "githubUrl": "https://github.com/Luffos/rxn-units",
+    "npmPkg": "rxn-units",
+    "examples": ["https://codesandbox.io/s/rxn-units-example-ox6n4h"],
+    "ios": true,
+    "android": true,
+    "web": true,
+    "expo": true,
+    "windows": true,
+    "macos": true,
+    "tvos": true
```

```suggestion
    "windows": true,
    "macos": true,
    "tvos": true
```

For out-of-tree platforms we require an example utilizing them, no matter if library uses native code ore not, since there are a lot of minor differences between the implementation of certain RN parts, which can lead to unexpected behavior.